tests/Fixer/ClassNotation/SingleClassElementPerStatementFixerTest.php

```diff
@@ -208,27 +208,21 @@ function doSomething()
 }
 EOT
             ),
-            array(
+            'line_breaks_1' => array(
                 <<<'EOT'
 <?php
 
 class Foo
 {
 
     public $bar = null;
-
```

Comment:
Because this fixer now uses [just the first previous line break](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1889/files#diff-fe9800e239d3b7335db6b921149d7a07R129) as separator, based on feedback from [Symfony](https://github.com/symfony/symfony/pull/19176#issuecomment-228517738).